setup.py

```diff
@@ -4,7 +4,7 @@
 
 def find_resource_files(directory):
     paths = []
-    for (path, directories, filenames) in os.walk(directory):
+    for (path, _directories, filenames) in os.walk(directory):
```

Comment:
If "_directories" is just not need, I'd stick with "_". Right now it looks like it's an internal variable to me.

Reply:
This applies to other cases "_dirs", "_i", etc.